extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -106,12 +114,20 @@ export function createTextInput(view: azdata.ModelView, inputInfo: { defaultValu
 	}).component();
 }
 
-export function createLabel(view: azdata.ModelView, info: { text: string, description?: string, required?: boolean, width?: string, fontStyle?: FontStyle, fontWeight?: FontWeight, links?: azdata.LinkArea[] }): azdata.TextComponent {
+export function createLabel(view: azdata.ModelView, info: { text: string, description?: string, required?: boolean, width?: string, fontStyle?: FontStyle, fontWeight?: FontWeight, links?: azdata.LinkArea[], color?: string, cssStyles?: CSSStyles }): azdata.TextComponent {
+	let cssStyles: CSSStyles = { 'font-style': info.fontStyle || 'normal', 'font-weight': info.fontWeight || 'normal'};
+	if (info.cssStyles !== undefined) {
+		cssStyles = Object.assign(info.cssStyles, cssStyles);
+	}
+
+	if (info.color !== undefined) {
+		cssStyles['color'] = info.color;
```

Comment:
If you're going to be passing in cssStyles as a general param anyways might as well just get rid of the specific style params and just combine them into the cssStyles param (fontStyle, fontWeight and color)

Reply:
I considered that. The idea is to have separate params for commonly used ones and use the cssStyles for the more esoteric ones. This makes for easier usage.

Reply:
At the cost of many more parameters and inconsistent handling of how styles are passed in. 

If you really want to make "common" styles more easily accessible you make make the parameter type have a set of predefined properties in addition to taking in extras such as 

`export type MyCssStyles {
    color?: string;
    fontWeight?: string;
    [key: string]: string;
}`

I'm fine with either - but both are better than having a mix of specified params and an object containing more.

Reply:
I like your suggestion of extending CSSStyles type to include named as well as generic properties to make this simpler and consistent. Will do the implementation accordingly.